cloud-sql/mysql/ConnectTcp.cs

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// [START cloud_sql_mysql_dotnet_ado_connect_tcp]
+using MySql.Data.MySqlClient;
+using System;
+
+namespace CloudSql
+{
+    public class MySqlTcp
+    {
+        public static MySqlConnectionStringBuilder NewMysqlTCPConnectionString()
+        {
+            // Equivalent connection string:
+            // "Uid=<DB_USER>;Pwd=<DB_PASS>;Host=<INSTANCE_HOST>;Database=<DB_NAME>;"
+            var connectionString = new MySqlConnectionStringBuilder()
+            {
+                // The Cloud SQL proxy provides encryption between the proxy and instance.
+                SslMode = MySqlSslMode.None,
+
+                // Remember - storing secrets in plain text is potentially unsafe. Consider using
```

Comment:
Let's update the text of this comment to match the Go sample:

```
// Note: Saving credentials in environment variables is convenient, but not
// secure - consider a more secure solution such as
// Cloud Secret Manager (https://cloud.google.com/secret-manager) to help
// keep secrets safe.
```

Reply:
Done.